extras/userdoc/md/documentation/part-2-populations-of-neurons.md

```diff
@@ -230,30 +270,46 @@ These are the new functions we introduced for the examples in this handout.
 
 ### Getting information about NEST
 
-`PrintNetwork(depth=1, subnet=None)`:  
```

Comment:
Same question as before: I don't think ``PrintNetwork`` is deprecated, only ``subnet``